zebra/rt_netlink.c

```diff
@@ -740,6 +740,16 @@ int netlink_route_change(struct nlmsghdr *h, ns_id_t ns_id, int startup)
 		return 0;
 	}
 
+	if (!(rtm->rtm_family == AF_INET || rtm->rtm_family == AF_INET6
+	      || rtm->rtm_family == AF_ETHERNET || rtm->rtm_family == AF_EVPN
+	      || rtm->rtm_family == AF_UNSPEC
+	      || rtm->rtm_family == AF_FLOWSPEC)) {
```

Comment:
I think we could add `AF_MPLS` here too (MPLS routes are expected and ignored on purpose a few lines below).

`AF_EVPN` and `AF_FLOWSPEC` are specific to FRR and don't need to be checked here. I believe `AF_UNSPEC` could be removed too but I'm not completely sure about it.